src/ports/postgres/modules/regress/marginal.py_in

```diff
@@ -522,9 +520,8 @@ def margins_mlogregr(schema_madlib, source_table, out_table,
             """.format(**all_arguments))
 
         # Rename the output summary table
-        rename_table(schema_madlib,
-                     "{old}_summary".format(old=mlog_out_table),
-                     "{new}_summary".format(new=out_table))
+        plpy.execute("""CREATE TABLE {out_table}_summary AS
```

Comment:
we should ideally drop the `mlog_out_table` table even if it is in the pg_temp schema. Otherwise it will be always be present in the current session.

Reply:
can we add a dev-check test to assert that the output summary table should exist ?